diff --git a/src/index.d.ts b/src/index.d.ts
index 228c1f0dbc3bbb5278288938f9fdbcff38e2c3f4..609d0c4d9ce32f5cca3633308c4ae5197ff87a8e 100644
--- a/src/index.d.ts
+++ b/src/index.d.ts
@@ -1,3 +1,14 @@
+export enum KittenVoice {
+  Bella = 'Bella',
+  Jasper = 'Jasper',
+  Luna = 'Luna',
+  Bruno = 'Bruno',
+  Rosie = 'Rosie',
+  Hugo = 'Hugo',
+  Kiki = 'Kiki',
+  Leo = 'Leo',
+}
+
 /**
  * kitten-tts-js — TypeScript declarations
  */
@@ -53,6 +64,10 @@ export declare class RawAudio {
 }

 export declare class KittenTTS {
+  static create(
+    modelId?: string,
+    opts?: ModelOptions
+  ): Promise<KittenTTS>;
   /**
    * Load a KittenTTS model from HuggingFace Hub.
    *
diff --git a/src/index.js b/src/index.js
index 0f558f6f0261d6aaf1c8c2f9ea45b3c912c57194..e67639074161125b8f0531cb49e757ccf8f920af 100644
--- a/src/index.js
+++ b/src/index.js
@@ -12,3 +12,14 @@ export { TextPreprocessor } from './preprocess.js';
 export { phonemize } from './phonemizer.js';
 export { loadNpz } from './npz-loader.js';
 export { downloadModel, MODELS } from './model-loader.js';
+
+export const KittenVoice = {
+  Bella: 'Bella',
+  Jasper: 'Jasper',
+  Luna: 'Luna',
+  Bruno: 'Bruno',
+  Rosie: 'Rosie',
+  Hugo: 'Hugo',
+  Kiki: 'Kiki',
+  Leo: 'Leo',
+};
diff --git a/src/kitten-tts.js b/src/kitten-tts.js
index 63e4d2a38c0840c48cb1b244ae7bee8f3ffec0bc..b41de50ee7bc4b97d3d339713b7e2d8a4506397e 100644
--- a/src/kitten-tts.js
+++ b/src/kitten-tts.js
@@ -54,6 +54,10 @@ export class KittenTTS {
    * @param {{ dtype?: string, cacheDir?: string }} [opts]
    * @returns {Promise<KittenTTS>}
    */
+  static async create(modelId = 'KittenML/kitten-tts-nano-0.8', opts = {}) {
+    return KittenTTS.from_pretrained(modelId, opts);
+  }
+
   static async from_pretrained(modelId = 'KittenML/kitten-tts-nano-0.8', opts = {}) {
     const { modelBuffer, voicesBuffer, config } = await downloadModel(modelId, opts);
